Code/GraphMol/MolDraw2D/DrawMol.cpp

```diff
@@ -2194,13 +2212,13 @@ int DrawMol::doesNoteClash(const DrawAnnotation &annot) const {
 
 // ****************************************************************************
 int DrawMol::doesRectClash(const StringRect &rect, double padding) const {
+  // No longer check if it clashes with highlights.  This frequently
+  // results in bad pictures and things look ok on top of highlights.
```

Comment:
There doesn't seem to be a code change associated with this comment

Reply:
I thought I had taken that out.  It has moved position in the function wrt the current master so perhaps another merge put it back in again.  It pertained to a previous PR and it seemed to me that unless someone had a knowledge of the history of the drawing code it wasn't very helpful.  I wonder if it would useful, however, to explain why we don't care if annotation clashes with highlights, in case in future someone tries to put that check in again.  In which case I will hunt down the relevant PR and reference it.

Reply:
ah, ok, that makes sense. let's leave it as is.